package.json

```diff
@@ -1,11 +1,29 @@
 {
-  "name": "todolist",
+  "name": "todo-list",
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
+  "private": "true",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "webpack"
```

Comment:
- [ ] I suggest that you add the script to start the server this one you have should be `build` that runs webpack you can use
`"start": "webpack serve --open"`,
 `"build": "webpack"`

Reply:
Hello @DeliceLydia, Thanks for giving me feedback. I am going to implement it.